File: test7C#/test7/Model/Donut/PowderedDonut.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace test7.Model.Donut
{
    class PowderedDonut : SimpleDonut
    {
        public PowderedDonut() : base("Powdered Donut")
        {
            base.Flavor("SUGAR!!!");
        }
    }
}

File: test7Go/oop/models/donut/PowderedDonut.go
package donut

type PowderedDonut struct {
	SimpleDonut
}

func NewPowderedDonut() PowderedDonut {
	simpleDonut := NewSimpleDonut("Powdered Donut")
	simpleDonut.Flavor = "SUGAR!!!"

	return PowderedDonut{simpleDonut}
}
